agent/containers/images/push

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+image_repo="${1}"
+distro="${2}"
+githash="$(grep -v -E "^v" ${distro}-tags.lis)"
+ver="$(grep -E "^v" ${distro}-tags.lis | grep -v latest)"
+
+if [[ "${3}" == "_major" ]]; then
```

Comment:
It'd be really helpful here to have comments showing the format of the `-tags.lis` files. I infer (the logic is dense) that these are generated using `gen-tags-from-rpm` in the format

```
<git-commit-sha>
v<version>
v<major-version>-latest v<major-version>.<minor-version>-latest
```

So `grep -v -E "^v"` returns the commit ID, the only line without a "v", while `grep -E "^v"`returns the full version line and the `_major` version line, distinguished by the `grep -v latest`.

I'd call this "elegant-ugly" ... a comment would really help the reader.